llvm/test/CodeGen/DirectX/length_invalid_intrinsic_error.ll

```diff
@@ -0,0 +1,10 @@
+; RUN: not opt -S -dxil-op-lower -mtriple=dxil-pc-shadermodel6.3-library %s 2>&1 | FileCheck %s
+
+; DXIL operation length does not support scalar types
+; CHECK: error: invalid intrinsic signature
+
+define noundef float @test_length_float(float noundef %p0) {
+entry:
+  %hlsl.length = call float @llvm.dx.length.f32(float %p0)
```

Comment:
We get this error for free from how we define the intrinsic.  I thought what we wanted an error test for was `@llvm.dx.length.v1f32`?

Reply:
Ok, I've added tests for both a scalar input, and a vector with 1 element input.